frontend_tests/node_tests/reactions.js

```diff
@@ -349,7 +349,7 @@ run_test("get_reaction_section", () => {
     const message_row = $.create("some-message-row");
     const message_reactions = $.create("our-reactions-section");
 
-    message_table.set_find_results("[zid='555']", message_row);
+    message_table.set_find_results(`[zid='${CSS.escape(555)}']`, message_row);
```

Comment:
Do we need `CSS.escape` in places where the tests are using string constants (as opposed to interpolating variables as we do in certain test helpers)?  I know this is a judgment call and I'll defer to Anders on this, but my vote would be to leave line 352 as is here.

Reply:
Yes, or the test fails. `CSS.escape(555)` is `"\\35 55"`, not `"555"`, because `CSS.escape` can also be used for CSS identifiers, which are not allowed to start with a digit. `[zid='555']` would be a functionally equivalent selector, but zjquery requires us to match the exact selector string generated by the code being tested, not just an equivalent one.

Reply:
Ugh, I guess there's no alternative to `CSS.escape` that converts "555" to "555"?